wiki/Modding/id.md

```diff
@@ -1,38 +1,40 @@
 
-**Kelima**, silahkan post mod yang sudah kita buat.
+Saat kamu menemukan masalah, pilih objek hit tersebut, lalu gunakan pintasan salin (`Ctrl` + `C`) untuk menyalin informasi dari timestamp sebagai teks. Pada diskusi beatmap, tempel (`Ctrl` + `V`) timestamp sekaligus umpan balik dari kamu. Tab `Timeline` digunakan untuk masalah yang berkaitan dengan bagian tertentu dari sebuah beatmap, tab `Umum (Tingkat Kesulitan ini)` digunakan untuk umpan balik pada bagian yang lebih besar dari sebuah beatmap, dan tab `Umum (Semua Tingkat Kesulitan)` digunakan untuk masalah yang berlaku untuk semua tingkat kesulitan pada sebuah beatmapset.
 
-## Hal-Hal Yang Perlu Diperhatikan Dalam Modding
+Semua postingan mod harus membahas apa saja masalahnya, mengapa itu menjadi masalah, dan bagaimana cara memperbaikinya. Lihat Kode etik dalam Modding dan Mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping?locale=id#membuat-postingan-mod) untuk detail lebih lanjut tentang cara membuat format postingan mod.
 
-- Baca 1 dan [2](https://osu.ppy.sh/community/forums/topics/439) sebelum kita melakukan modding untuk pertama kali.
-- Hal yang paling penting dan kerap dilupakan dalam modding adalah menyertakan alasan kepada setiap hal yang kita sebutkan. Ingat, tujuan kita memoding adalah membantu mapper menjadi lebih baik dalam mapnya. Jika kita tidak menyertakan alasan, biasanya mapper akan mengabaikan saran kita, dan kemungkinan besar tidak akan mengerti apa yang kita utarakan.
-- Hal yang paling mudah dan paling sering dilakukan pertama kali adalah mencek kesalahan pada folder lagu, seperti bitrate mp3, ukuran gambar, ukuran filesize, dan file-file yang seharusnya tidak termasuk di dalam folder lagu.
-- Perlu di ingat bahwa ukuran bitrate mp3 harus diantara 128kbps-192kbps, ukuran gambar sebaiknya di antara 800x600-1024x780. Kita juga bisa memberitahu mapper untuk mengcompress gambar jika ada file gambar yang ukurannya cukup besar (sekitar 500kb-1mb)
-- Setelah folder check, hal yang paling sering dilakukan adalah mencek kesalahan pada info field lagu. Misalnya nama artis, judul lagu, source, dan tags.
-- Ingat bahwa semua info field pada lagu harus sama. Dan beritahu mapper jika mapper mengulang artis dan source pada tag.
-- Setelah itu hal yang paling mudah dilakukan adalah mengecek konsistensi timing, seperti BPM dan offset, pada tiap tingkat kesulitan.
+Beatmapper biasanya memberikan [kudosu](/wiki/Glossary#kudosu!) ke postingan mod yang mereka anggap berguna dengan mengeklik tombol *jempol*. Statistik ini dapat dilacak pada profil setiap pengguna dan menjadi rujukan saat melamar untuk bergabung dengan [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators).
```

Comment:
```suggestion
Beatmapper biasanya memberikan [kudosu](/wiki/Glossary#kudosu!) ke postingan mod yang mereka anggap berguna dengan mengeklik tombol ibu jari. Statistik ini dapat dilacak pada profil setiap pengguna dan menjadi rujukan saat melamar untuk bergabung dengan [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators).
```
dunno why is there the need to use jempol tbh xD

Reply:
wkwkwkwk sebenernya sama aja sih dua2nya